include/dlaf/solver/triangular/mc/triangular_LLN.h

```diff
@@ -129,26 +129,27 @@ void triangular_LLN(comm::CommunicatorGrid grid, blas::Diag diag, T alpha,
         // Avoid useless communication if one-column communicator
         if (row_comm_size > 1) {
           hpx::dataflow(executor_mpi, hpx::util::unwrapping([](auto&& tile, auto&& comm_wrapper) {
-                          comm::sync::broadcast::send(comm_wrapper().rowCommunicator(), tile);
+                          comm::sync::broadcast::send(comm_wrapper.ref().rowCommunicator(), tile);
                         }),
                         mat_a.read(kk), serial_comm());
         }
         kk_tile = mat_a.read(kk);
       }
       else {
         if (row_comm_size > 1) {
-          kk_tile = hpx::dataflow(executor_mpi,
-                                  hpx::util::unwrapping([](auto index, auto&& tile_size,
-                                                           auto&& comm_wrapper) -> ConstTileType {
-                                    memory::MemoryView<T, Device::CPU> mem_view(
-                                        util::size_t::mul(tile_size.rows(), tile_size.cols()));
-                                    TileType tile(tile_size, std::move(mem_view), tile_size.rows());
-                                    comm::sync::broadcast::receive_from(index,
-                                                                        comm_wrapper().rowCommunicator(),
-                                                                        tile);
-                                    return std::move(tile);
-                                  }),
-                                  k_rank_col, mat_a.tileSize(GlobalTileIndex(k, k)), serial_comm());
+          kk_tile =
+              hpx::dataflow(executor_mpi,
+                            hpx::util::unwrapping([](auto index, auto&& tile_size,
+                                                     auto&& comm_wrapper) -> ConstTileType {
+                              memory::MemoryView<T, Device::CPU> mem_view(
+                                  util::size_t::mul(tile_size.rows(), tile_size.cols()));
+                              TileType tile(tile_size, std::move(mem_view), tile_size.rows());
+                              comm::sync::broadcast::receive_from(index,
+                                                                  comm_wrapper.ref().rowCommunicator(),
+                                                                  tile);
+                              return std::move(tile);
+                            }),
+                            k_rank_col, mat_a.tileSize(GlobalTileIndex(k, k)), serial_comm());
```

Comment:
Would it not make sense to name this function as well, and the ones further down in the file (apologies if you already planned to do this)?

Reply:
Yes, definitely, I am preparing a separate PR for `triangular_LLN()`. I had to touch it because I made changes to `Pipeline`.